sui-execution/latest/sui-adapter/src/programmable_transactions/execution.rs

```diff
@@ -951,6 +993,7 @@ mod checked {
                 vec![],
                 vec![],
                 /* is_init */ true,
+                None, // TODO: add tracing for init functions?
```

Comment:
Is there a reason not to do this to begin with?

Reply:
I was thinking about it from the trace-debugging perspective (it was not super clear to me how/where/when to visualize these as part of a PTB execution flow) but these traces obviously do not have to be consumed by the trace debugger only, so they should indeed be generated here (even we decide for the trace debugger to ignore them initially).